composer.json

```
@@ -77,5 +77,59 @@
             "ergebnis/composer-normalize": true
         },
         "sort-packages": true
+    },
+    "scripts": {
+        "cs:check": "php php-cs-fixer fix --dry-run --diff",
```

```suggestion
        "cs:check": "@php php-cs-fixer fix --dry-run --diff",
```
here and all places below, see https://getcomposer.org/doc/articles/scripts.md#executing-php-scripts